scrapers/Mandyflores.yml

```diff
@@ -0,0 +1,29 @@
+name: Mandyflores
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - mandyflores.com/vod/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $updateDesc: //span[@class="update_description"]
+      $divCenter: /div[@align="center"]
+    scene:
+      Title: //span[@class="title_bar_hilite"]
+      Details:
+        selector: $updateDesc$divCenter/span/span[@style]/text()[normalize-space(.)]|($updateDesc | $updateDesc/p)/text()[normalize-space(.)]|($updateDesc$divCenter/text())[1]|($updateDesc/text())[1]
+        concat: "\n"

```

Comment:
Should we switch that to `concat: "\n\n"` ? Imho it maintains better the original structure (sample `https://mandyflores.com/vod/scenes/Ill-Tell-Mom_vids.html`)